internal/sql/repository/team/TeamRepository.go

```diff
@@ -57,7 +57,7 @@ func (impl TeamRepositoryImpl) Save(team *Team) error {
 
 func (impl TeamRepositoryImpl) FindAll() ([]Team, error) {
```

Comment:
change name of function it is FindAllActive

Reply:
done